cli/util/yaml.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type ProjectConfig struct {
```

Comment:
I had a lot of trouble figuring out how to parse YAML without defining it's struct as part of the initial object. It ended up feeling cleaner & easier to just pour all the YAML handling into the new project-config.go